core/runtime/src/test/java/org/eclipse/sw360/antenna/util/TemplateRendererTest.java

```diff
@@ -100,7 +101,7 @@ private String wrapTemplateXml(String innerTemplate) {
 
     private File writeTemplateFromString(String template) throws IOException {
         final File templateFile = folder.newFile("template.xml");
-        FileUtils.writeStringToFile(templateFile, template, StandardCharsets.UTF_8);
+        Files.write(templateFile.toPath(), template.getBytes(StandardCharsets.UTF_8));
```

Comment:
`templateFile` (and thus the parent directory) already exists due to `newFile` in line 103.